extension/test/test-logger.js

```diff
@@ -10,7 +10,13 @@ var verifyAsyncOutput = function (test, logger, testFunc) {
   var message = '';
 
   return function (status) {
-    var line = {}, hasMore;
+    var line = {}, hasMore, isSuccessStatus;
+
+    isSuccessStatus = typeof status == "number";
+    test.assert(isSuccessStatus, status["message"] ? status.message : status);
+    if (!isSuccessStatus) {
+      return;
+    }
```

Comment:
:)

This PR is merged!
Just wanted to file a new issue with title "Update tests to use `test/assert` module instead the deprecated `TestRunner`".

Sure, this could be fold into #219, but it could have it's own issue.
I vote for the latter.